lib/common/tests/options/pcmk__env_option_test.c

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2022 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#include <crm_internal.h>
+#include "mock_private.h"
+
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+#include <setjmp.h>
+#include <cmocka.h>
```

Comment:
Once #2798 gets merged, you can reduce much of this block to `#include <crm/common/unittest_internal.h>` - see [8d8dbd4](https://github.com/ClusterLabs/pacemaker/pull/2798/commits/8d8dbd466325f836f91b6bfd262c727cb1191144).